pkg/util/metricsinfo/metrics_info.go

```diff
@@ -94,11 +94,16 @@ type QueryNodeConfiguration struct {
 	SimdType string `json:"simd_type"`
 }
 
+type QueryNodeCollectionMetrics struct {
+	Collections map[int64]int64
```

Comment:
Unclear naming, better to rename to collectionRows or anything else

Reply:
ok